dev_guide/managing_images.adoc

```diff
@@ -1239,3 +1240,7 @@ for the whole cluster.
 <8> Languages this image supports. This value will be used during `oc new-app` invocations to try to match potential
 builder images to the provided source repository.
 <9> Version information for this tag.
+<10> The type of object this `ImageStreamTag` is referencing.  Valid values are: `DockerImage`, `ImageStreamTag`, and `ImageStreamImage`.
+<11> The object this `ImageStreamTag` will import.
+
+For more information on the fields that can be defined in an `ImageStream`, see the xref:../rest_api/apis-image.openshift.io/v1.ImageStream.adoc#object-schema[Imagestream API] and the xref:../rest_api/apis-image.openshift.io/v1.ImageStreamTag.adoc#object-schema[ImagestreamTag API].
```

Comment:
@mburke5678 this link to the REST API doesn't work on 3.6 and below branches as the REST API changed to different structure from 3.7 onwards. I have fixed it via: https://github.com/openshift/openshift-docs/pull/9965 and below.